playbook.yml

```diff
@@ -7,6 +7,9 @@
   - include_role:
       name: "couchbase-infra"
     when: couchbase.servers.size is defined and couchbase.servers.size > 0
+  - include_role:
+      name: "couchbase-infra"
+    when: couchbase is defined and couchbase.servers.size > 0
```

Comment:
I see the general idea shaping is to use the parent dictionary for the 'is defined' check, which is fine. This currently has the couchbase-infra include_role: task in twice, so that redundancy should probably be cleaned up before merging. Also, this copy of playbook.yml is checking uperf.pair instead of uperf for that role, and at master that's already changed to uperf.